sbt.hcl

```diff
@@ -0,0 +1,18 @@
+description = "sbt is a build tool for Scala, Java, and more"
+homepage = "https://www.scala-sbt.org/"
+binaries = ["sbt/bin/sbt"]
+test = "sbt --version"
+
```

Comment:
You might want to add
```
requires = ["jre"]
```
to make sure there is a JRE present, like in `gradle.hcl`